src/libraries/System.Windows.Extensions/src/CompatibilitySuppressions.xml

```diff
@@ -1,12 +1,11 @@
-﻿<?xml version="1.0"?>
+﻿<?xml version="1.0" encoding="utf-8"?>
 <Suppressions xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xmlns:xsd="http://www.w3.org/2001/XMLSchema">
-  <!-- .NETCoreapp,Version=v3.0 is out of support and dropped from the package. -->
   <Suppression>
     <DiagnosticId>PKV006</DiagnosticId>
-    <Target>.NETCoreApp,Version=v3.0</Target>
+    <Target>.NETCoreApp,Version=v3.1</Target>
```

Comment:
Should we add the comment on top back but explain that 3.1 was dropped this time?

Reply:
Sure, I can do that.